_posts/0000-01-02-kharriscreates1.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
You're killing me smalls
![1_4691735](https://user-images.githubusercontent.com/73741755/97778186-800c9880-1b43-11eb-9678-4e0c612d3f10.png)